facilitator/src/bin/facilitator.rs

```diff
@@ -539,6 +542,50 @@ fn main() -> Result<(), anyhow::Error> {
                 .add_packet_decryption_key_argument()
                 .add_batch_signing_key_arguments()
         )
+        .subcommand(
+            SubCommand::with_name("lint-manifest")
+            .about("Validate and print out global or specific manifests")
+            .arg(
+                Arg::with_name("manifest-base-url")
+                    .long("manifest-base-url")
+                    .value_name("URL")
+                    .help("base URL relative to which manifests may be fetched")
+                    .long_help(
+                        "base URL relative to which manifests may be fetched \
+                        over HTTPS. Should be in the form \"https://foo.com\"."
+                    )
+                    .required_unless("manifest-path")
+            )
+            .arg(
+                Arg::with_name("manifest-path")
+                    .long("manifest-path")
+                    .value_name("PATH")
+                    .help("path to local manifest file to lint")
+                    .required_unless("manifest-base-url")
+            )
+            .arg(
+                Arg::with_name("manifest-kind")
```

Comment:
Since all these flag names are scoped to the `lint-manifest` subcommand, let's drop the `manifest-` prefix and save some typing.